src/auth/authApp.js

```diff
@@ -27,25 +27,26 @@ const DEFAULT_PROFILE = {
 }
 
 /**
- * Generates a ECDSA keypair and stores the hex value of the private key in
- * local storage.
+ * Fetches the hex value of the transit private key from local storage.
  * @return {String} the hex encoded private key
  * @private
  */
-export function generateAndStoreTransitKey() {
-  const transitKey = makeECPrivateKey()
-  localStorage.setItem(BLOCKSTACK_APP_PRIVATE_KEY_LABEL, transitKey)
-  return transitKey
+export function getTransitKey() : string {
+  const transitKey = localStorage.getItem(BLOCKSTACK_APP_PRIVATE_KEY_LABEL)
+  return ((transitKey: any): string)
 }
 
 /**
- * Fetches the hex value of the transit private key from local storage.
+ * Generates a ECDSA keypair to
+ * use as the ephemeral app transit private key
```

Comment:
Maybe just "transit private key" which is the phrase you used for me. We can explain that a transit private key is ephemeral elsewhere.